test/Performance/setup_env_unix.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+set -e
+if [[ ("$1" = "Ubuntu16" || "$1" = "RedHat7" || "$1" = "Sierra") ]]; then
+    PLATFORM=$1
+else
+    echo "First argument must be one of Ubuntu16, RedHat7, Sierra. Exiting..."
+    exit 1
+fi
+if [[ "$2" = "" ]]; then
```

Comment:
I mean why doesn't Windows have a check to exit if the provided PHP version is not 7.x.x